Update deploy-a-remix-site.md (#8356)
cloudflare/cloudflare-docs: content/pages/framework-guides/deploy-a-remix-site.md

```

> After running the above command, you will be prompted to enter a path for your project in your terminal (_Where would you like to create your app?_), and an option to select your deploy method (_What type of app do you want to create?_). This guide uses Cloudflare Pages, so select "Just the basics" (it's default, so just hit enter), and then use the down arrow to highlight Cloudflare Pages and hit enter to select.

Reviewer: ```suggestion
After running the above command, you will be prompted to answer configuration questions and select your deploy method. This guide uses Cloudflare Pages.
```
